test/simpleGaMultiSigTest.js

```diff
@@ -38,45 +45,40 @@ describe('SimpleGAMultiSig', () => {
   }
   let consensusInfo;
 
-  const getTxHash = (rlpTransaction) => new Uint8Array(Crypto.hash(Buffer.concat([
-    Buffer.from(aeSdk.getNetworkId()),
-    TxBuilderHelper.decode(rlpTransaction, 'tx'),
+  const getTxHash = async (rlpTransaction) => new Uint8Array(hash(Buffer.concat([
+    Buffer.from(await aeSdk.getNetworkId()),
```

Comment:
@davidyuk wondering why this is async now. I think that wasn't mentioned in the migration guide. not 100% sure though

Reply:
All data that can be returned from node is async now because we have the node initialisation sync instead. So before sdk was getting network id in async node initialisation, and now we have to await it separatelly, but we can just write `new Node(...)` (without await).
https://github.com/aeternity/aepp-sdk-js/blob/0478eed0a86b959e62b8b63a261711ec18888d90/docs/guides/migration/12.0.0.md#getnetworkid-returns-a-promise